FinderKeeper/Assets/Mapbox/Core/Plugins/ThirdParty/Mapbox.IO.Compression/net35/Mapbox.IO.Compression.pdb.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
I've noticed these Mapbox.IO.Compression changes popping up, we'll have to keep an eye on them